subprojects/core/src/test/groovy/org/gradle/internal/classpath/DefaultCachedClasspathTransformerTest.groovy

```diff
@@ -409,10 +409,10 @@ class DefaultCachedClasspathTransformerTest extends ConcurrentSpec {
         def file = testDir.file("thing.jar")
         jar(file)
         def classpath = DefaultClassPath.of(file)
-        def cachedFile = testDir.file("cached/8bccc8c51683456910b894e02dc7e915/thing.jar")
+        def cachedFile = testDir.file("cached/bc4f3e05266cfadaa3cd1f4484942739/thing.jar")
```

Comment:
🤔 Wonder how we could get rid of hashes like this in test code that need to be updated after some changes...

Reply:
Yes, this is annoying. I think when we remove all instrumentation with this custom cached classpath transformer we could also remove some if these test here and maybe get rid of that

Reply:
That's a good point. Let's make sure that we do remove them once we can, though my understanding is that we can't do it after merging this PR, right?

Reply:
Yes, that is right. We need to clean up two more places to remove it

Reply:
Actually looking at this test it looks like we will keep it even after we remove all custom classpath transformations, since we use that class also in other cases. 

Do you feel it's worth finding another solution now, or it can wait for another occasion?

Reply:
Nah, just annoyed that we have it. I think we done enough for now, let's think of a solution the next time we need to change something in this area.